modules/ossm-threescale-applying-external-service-entry-objects.adoc

```diff
@@ -49,25 +96,52 @@ spec:
   location: MESH_EXTERNAL
   resolution: DNS
 ----
+.. Add the `DestinationRule` CR to a file called _destination-rule-threescale-saas-system.yml:_
```

Comment:
I recommend using back-ticks for the file name references, instead of italicising them. This is consistent with other examples in the docs.